### PR TITLE
Feature/3576 javascript schema support for the cli

### PR DESCRIPTION
**Pull request by MHove89**

### CODE REVIEW

File: src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs [hunk 1]
```diff
@@ -23,7 +27,7 @@ public SchemaFileService(ILogger<SchemaFileService> logger)
         _logger = logger;
     }
 
-    public void CreateSchema(string alias, SchemaType schemaType, string content = null)
+    public void CreateSchema(string alias, SchemaType schemaType, MappingSchemaVersion version = null)
```
Comment (jesperweber) [inline src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs:30]: The CreateSchema method does not create empty schemas locally, like the old version, which means the the "schema create" command is failing in my test.

File: src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs [hunk 2]
```diff
@@ -42,6 +42,7 @@ public new class Handler : BaseCommandHandler, ICommandHandler
             public string Alias { get; set; }
             public string Name { get; set; }
             public string Type { get; set; }
+            public string Format { get; set; }
```
Comment (jesperweber) [inline src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs:45]: There is not command option to set the schema format

File: src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs (hunk 1 above)
Comment (MHove89) [reply src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs]: This has been worked on. Could you verify that it works?

File: src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs (hunk 2 above)
Comment (MHove89) [reply src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs:46]: This has been added. Could you verify that it works?

File: src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs (hunk 1 above)
Comment (jesperweber) [reply src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs]: Perfect, it's working.

File: src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs (hunk 2 above)
Comment (jesperweber) [reply src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs:46]: Yes, it works now.

**Review by jesperweber — APPROVED** [review]

**Pull request merged**